lib/elastic_record/connection.rb

```diff
@@ -14,7 +14,7 @@ def initialize(servers, options = {})
       self.request_count      = 0
       self.max_request_count  = 100
       self.options            = options.symbolize_keys
-      self.bulk_actions       = nil
+      self.bulk_actions       = {}
```

Comment:
Instead of this, try

```ruby
Thread.current['bulk_actions'] = nil
```

Look at this project for other uses of `Thread.current`